open_optical_gating/cli/optical_gater_server.py

```diff
@@ -193,9 +198,6 @@ def sync_state(self, frame):
             / self.pog_settings["reference_period"]
         )  # rad
 
-        # Gets the current timestamp in seconds
-        current_time_s = time.time() - self.initial_process_time_s
-
         # Calculate cumulative phase (phase) from delta phase (current_phase - last_phase)
         if self.frame_num == 0:
```

Comment:
self.frame_num appears in a few different places. Are you happy that it's as consistent and clear as it can be, in terms of who resets it to 0 and when/where?